package-lock.json

```diff
@@ -1,6 +0,0 @@
-{
-  "name": "opensearch-api-specification",
-  "lockfileVersion": 3,
-  "requires": true,
-  "packages": {}
-}
```

Comment:
Should we be running an `npm install` to lock dependencies? Or exclude this file in .gitignore. Either way probably a separate PR.